vlog4j-client/src/main/java/org/semanticweb/vlog4j/client/picocli/VLog4jClientMaterialize.java

```diff
@@ -230,8 +230,7 @@ private void doSaveQueryResults(final Reasoner reasoner, final PositiveLiteral q
 	}
 
 	private void doPrintResults(final Reasoner reasoner, final PositiveLiteral query) {
-		System.out.println(
-				"Number of query answers in " + query + ": " + ClientUtils.getQueryAnswerCount(query, reasoner));
+		System.out.println("Number of query answers in " + query + ": " + reasoner.queryAnswerSize(query));
```

Comment:
now, this will print what `toString()` value of the returned `QueryAnswersSize` object. As `toString()` is not overwritten in `QueryAnswersSize` class, the default implementation from `Object` class will be called, giving an  undreadable hash code.
I suggest to print `reasoner.queryAnswerSize(query).getSize()` and `reasoner.queryAnswerSize(query).getCorrectness()`  here.

 `QueryAnswersSize` should also overwrite `toString `to provide information about `size `and also `correctness`.